Tensile/Source/lib/include/Tensile/ContractionProblem.hpp

```diff
@@ -588,8 +588,16 @@ namespace Tensile
         using BetaType  = Beta;
 
         TypedContractionInputs();
-        TypedContractionInputs(
-            A const* _a, B const* _b, C const* _c, D* _d, Alpha _alpha, Beta _beta);
+        TypedContractionInputs(A const* _a,
+                               B const* _b,
+                               C const* _c,
+                               D*       _d,
+                               Alpha    _alpha,
+                               Beta     _beta,
+                               size_t   _aElements,
```

Comment:
Adding these arguments is a fundamental change to the interface to Tensile.  We can't do this just for internal debugging purposes.